src/main/java/com/bootcamp/demo/data_access/AccountRepository.java

```diff
@@ -28,4 +28,13 @@ public String add(Account account) throws ExecutionException, InterruptedExcepti
         });
         return futureTransaction.get();
     }
-}
+
+    @Override
+    public String updatePassword(String username, String newPassword) throws ExecutionException, InterruptedException {
```

Comment:
should password change process require for old password? or maybe a confirmation of the new password?